### PR TITLE
Update Socket

### PR DESCRIPTION
**Pull request by chinazor-allen**
#### What does this PR do?
updates socket to current version

### CODE REVIEW

File: test/start-transcription-job.test.js [hunk 1]
```diff
@@ -18,7 +18,6 @@ const args = {
 describe('start-transcription-job', () => {
   it('should start a transcription job if valid parameters supplied', async () => {
     const { data: transcribeJob, code } = await run('start-transcription-job', { args, config });
-    expect(code).to.equal(200);
```
Comment (adamwardecki) [inline test/start-transcription-job.test.js]: why remove this check? If the reason is valid you can remove `code` from the above const assignment

File: package.json [hunk 2]
```diff
@@ -14,7 +14,6 @@
     "lint": "esw src test"
   },
   "dependencies": {
-    "@syncano/cli": "^0.9.3",
```
Comment (adamwardecki) [inline package.json:17]: why remove the `@syncano/cli` package?

File: socket.yml [hunk 3]
```diff
@@ -3,19 +3,20 @@ description: Amazon transcribe Syncano Socket
 long_description: |
   Visit link to know more about Amazon Transcribe automatic speech recognition
   https://docs.aws.amazon.com/transcribe/latest/dg/what-is-transcribe.html
-version: 0.0.3
+version: 0.0.1
```
Comment (adamwardecki) [inline socket.yml]: Why did you decrease the socket.yml version? It should be bumped automatically after successful test runs during Circle CI deployment

**Review by adamwardecki — CHANGES_REQUESTED** [review]

File: test/start-transcription-job.test.js (hunk 1 above)
Comment (chinazor-allen) [reply test/start-transcription-job.test.js]: This was not intentional. I probably deleted it during cleanup

File: socket.yml (hunk 3 above)
Comment (chinazor-allen) [reply socket.yml]: Oh I see. Fixed

File: test/start-transcription-job.test.js (hunk 1 above)
Comment (chinazor-allen) [reply test/start-transcription-job.test.js]: fixed

File: package.json (hunk 2 above)
Comment (chinazor-allen) [reply package.json:17]: Had a previous understanding that it should be globally installed, but this is fixed now

**Comment by chinazor-allen** [comment]
All done

**Review by adamwardecki — APPROVED** [review]

**Pull request merged**